packages/contracts/contracts/default-apps/DefaultableProxy.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity 0.5.10;
+pragma experimental "ABIEncoderV2";
+
+import "../interfaces/CounterfactualApp.sol";
+import "../ChallengeRegistry.sol";
+
+/*
+Before `switchesOutcomeAt`, this contract should return the exact
+same outcome as the outcome `targetAppIdentityHash` returns. However,
+after `switchesOutcomeAt`, it should return `defaultOutcome`.
+
+`challengeRegistryAddress` is used to look up the outcome.
+*/
+contract DefaultableProxy {
```

Comment:
name is confusing... maybe TimeDependentOutcome or something

Reply:
`TimeDependentOutcome` is too general, it includes every function which examines `block.number`, but this does so in a specific way

Reply:
Maybe `TimeLockedPassthrough`?